config_generator.py

```diff
@@ -59,10 +64,13 @@ def request_authentication():
     :return: None
     """
     try:
-        choice = input('Do you wish to use SUSI in Authenticated Mode? (y/n)\n')
+        # choice = input('Do you wish to use SUSI in Authenticated Mode? (y/n)\n')
```

Comment:
@stealthanthrax looks good.

But taking directly as a user input through `input` might be a better choice than going for command line inputs.
If the order of the inputs get changes?

Reply:
@prateekiiest , this will serve as an API endpoint so asking an input from the user would not be possible

Reply:
ok, got it :+1: